app/views/foreman_tasks/recurring_logics/index.html.erb

```diff
@@ -62,6 +63,7 @@
       <td><%= format_recurring_logic_limit recurring_logic.end_time.try(:in_time_zone) %></td>
       <td><%= recurring_logic_state(recurring_logic) %></td>
       <td><%= recurring_logic_action_buttons(recurring_logic) %></td>
+      <td><%= recurring_logic.purpose %></td>
```

Comment:
This needs to be one line higher
![image](https://user-images.githubusercontent.com/7326770/130582454-f556b7ee-7140-4c78-a9fb-7ae8b35673e4.png)